cmd/operator-opamp-bridge/config/config.go

```diff
@@ -269,7 +269,7 @@ func LoadFromFile(cfg *Config, configFile string) error {
 	if err != nil {
 		return err
 	}
-	if err = yaml.UnmarshalStrict(yamlFile, cfg); err != nil {
+	if err = yaml.Unmarshal(yamlFile, cfg); err != nil {
```

Comment:
What about this new feature required the function switch?

Reply:
nothing, but this is something i noticed us doing that we probably shouldn't be. See [this PR for more](https://github.com/open-telemetry/opentelemetry-operator/pull/2466). TL;DR: doing this will allow us to be more flexible with the supplied configuration from the operator in the future.